assets/stylesheets/shared/functions/_z-index.scss

```diff
@@ -141,6 +141,7 @@ $z-layers: (
 		'.notices-list.is-pinned': 180,
 		'.notices-list.is-pinned .notice': 180,
 		'.masterbar': 180,
+		'.gdpr-banner': 185,
```

Comment:
Is there a risk of it being "hidden" by blocks that have a higher `z-index`. It can be super frustrating to have that kind of banner not "clickable"

Reply:
I think this value should be a good compromise, but thanks for double-checking! It will be below a few other things but from what I can tell those shouldn't pose a problem.